### PR TITLE
Change branding in .github/ YAML files

### PR DESCRIPTION
**Pull request by TheOPtimal**
- Change issue templates to Prism branding
- Remove Matrix room link until one is created
- Remove PolyMC's OpenCollective

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -8,9 +8,9 @@ body:
       If you need help with running Minecraft, please visit us on our Discord before making a bug report.
 
       Before submitting a bug report, please make sure you have read this *entire* form, and that:
-      * You have read the [PolyMC wiki](https://polymc.org/wiki/) and it has not answered your question.
+      * You have read the [PrismLauncher wiki](https://polymc.org/wiki/) and it has not answered your question.
```
Comment (DioEgizio) [inline .github/ISSUE_TEMPLATE/bug_report.yml]: we'd update the link once it's up

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -25,15 +25,15 @@ body:
     - Other
 - type: textarea
   attributes:
-    label: Version of PolyMC
-    description: The version of PolyMC used in the bug report.
-    placeholder: PolyMC 1.4.1
+    label: Version of PrismLauncher
+    description: The version of PrismLauncher used in the bug report.
+    placeholder: PrismLauncher 1.4.1
```
Comment (DioEgizio) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
    placeholder: Prism Launcher 5.0
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -25,15 +25,15 @@ body:
     - Other
 - type: textarea
   attributes:
-    label: Version of PolyMC
-    description: The version of PolyMC used in the bug report.
-    placeholder: PolyMC 1.4.1
+    label: Version of PrismLauncher
+    description: The version of PrismLauncher used in the bug report.
```
Comment (DioEgizio) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
    description: The version of Prism Launcher used in the bug report.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -25,15 +25,15 @@ body:
     - Other
 - type: textarea
   attributes:
-    label: Version of PolyMC
-    description: The version of PolyMC used in the bug report.
-    placeholder: PolyMC 1.4.1
+    label: Version of PrismLauncher
```
Comment (DioEgizio) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
    label: Version of Prism Launcher
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -25,15 +25,15 @@ body:
...
+    description: The version of PrismLauncher used in the bug report.
+    placeholder: PrismLauncher 1.4.1
   validations:
     required: true
 - type: textarea
   attributes:
     label: Version of Qt
-    description: The version of Qt used in the bug report. You can find it in Help -> About PolyMC -> About Qt.
+    description: The version of Qt used in the bug report. You can find it in Help -> About PrismLauncher -> About Qt.
```
Comment (DioEgizio) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
    description: The version of Qt used in the bug report. You can find it in Help -> About Prism Launcher -> About Qt.
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 6]
```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: true
-contact_links:
-  - name: PolyMC Matrix Support Room
-    url: https://matrix.to/#/#support:polymc.org
-    about: Please ask for support here before opening an issue.
+# contact_links:
+#   - name: PrismLauncher Matrix Support Room
```
Comment (DioEgizio) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
#   - name: Prism Launcher Matrix Support Room
```

File: .github/ISSUE_TEMPLATE/rfc.yml [hunk 7]
```diff
@@ -6,7 +6,7 @@ body:
 - type: markdown
   attributes:
     value: |
-      ### Use this form to suggest a larger change for PolyMC.
+      ### Use this form to suggest a larger change for PrismLauncher.
```
Comment (DioEgizio) [inline .github/ISSUE_TEMPLATE/rfc.yml]:
```suggestion
      ### Use this form to suggest a larger change for Prism Launcher.
```

File: .github/ISSUE_TEMPLATE/rfc.yml [hunk 8]
```diff
@@ -18,7 +18,7 @@ body:
   attributes:
     label: Motivation
     description: |
-      Introduce the topic. If this is a not-well-known section of PolyMC, a detailed explanation of the background is recommended.
+      Introduce the topic. If this is a not-well-known section of PrismLauncher, a detailed explanation of the background is recommended.
```
Comment (DioEgizio) [inline .github/ISSUE_TEMPLATE/rfc.yml]:
```suggestion
      Introduce the topic. If this is a not-well-known section of Prism Launcher, a detailed explanation of the background is recommended.
```

File: .github/ISSUE_TEMPLATE/suggestion.yml [hunk 9]
```diff
@@ -5,25 +5,25 @@ body:
 - type: markdown
   attributes:
     value: |
-      ### Use this form to suggest a feature for PolyMC.
+      ### Use this form to suggest a feature for PrismLauncher.
```
Comment (DioEgizio) [inline .github/ISSUE_TEMPLATE/suggestion.yml]:
```suggestion
      ### Use this form to suggest a feature for Prism Launcher.
```

File: .github/ISSUE_TEMPLATE/suggestion.yml [hunk 10]
```diff
@@ -5,25 +5,25 @@ body:
...
 - type: input
   attributes:
     label: Role
-    description: In what way do you use PolyMC that needs this feature?
+    description: In what way do you use PrismLauncher that needs this feature?
```
Comment (DioEgizio) [inline .github/ISSUE_TEMPLATE/suggestion.yml]:
```suggestion
    description: In what way do you use Prism Launcher that needs this feature?
```

File: .github/ISSUE_TEMPLATE/suggestion.yml [hunk 11]
```diff
@@ -5,25 +5,25 @@ body:
...
     placeholder: I play modded Minecraft.
   validations:
     required: true
 - type: input
   attributes:
     label: Suggestion
-    description: What do you want PolyMC to do?
+    description: What do you want PrismLauncher to do?
```
Comment (DioEgizio) [inline .github/ISSUE_TEMPLATE/suggestion.yml]:
```suggestion
    description: What do you want Prism Launcher to do?
```

File: .github/ISSUE_TEMPLATE/suggestion.yml [hunk 12]
```diff
@@ -5,25 +5,25 @@ body:
...
     placeholder: I want the cat button to meow.
   validations:
     required: true
 - type: input
   attributes:
     label: Benefit
-    description: Why do you need PolyMC to do this?
+    description: Why do you need PrismLauncher to do this?
```
Comment (DioEgizio) [inline .github/ISSUE_TEMPLATE/suggestion.yml]:
```suggestion
    description: Why do you need Prism Launcher to do this?
```

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (DioEgizio) [reply .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
      * You have read the [Prism Launcher wiki](https://polymc.org/wiki/) and it has not answered your question.
```

also

File: .github/FUNDING.yml [hunk 13]
```diff
@@ -1 +1,2 @@
-open_collective: polymc
+# Commented out until further notice
+# open_collective: polymc
```
Comment (RuiNtD) [inline .github/FUNDING.yml]: We have https://opencollective.com/prismlauncher
Even if you can't donate yet, might as well add the link for when it's ready.
```suggestion
open_collective: prismlauncher
```

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (RuiNtD) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Updated link
```suggestion
      * You have read the [Prism Launcher wiki](https://prismlauncher.org/wiki/) and it has not answered your question.
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 14]
```diff
@@ -1,5 +1,5 @@
...
+#     url: about:blank # Add proper link in the future
+#     about: (⚠ Room not created yet ⚠) Please ask for support here before opening an issue.
```
Comment (RuiNtD) [inline .github/ISSUE_TEMPLATE/config.yml]: Add a link to the Matrix support room:
```suggestion
contact_links:
  - name: Prism Launcher Matrix Support Room
    url: https://matrix.to/#/#prism-support:matrix.org
    about: Please ask for support here before opening an issue.
```

**Review by RuiNtD — CHANGES_REQUESTED** [review]
Along with @DioEgizio's changes, we should go ahead and add these links to Prism Launcher's wiki, Open Collective, and Matrix support room.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 15]
```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: true
-contact_links:
-  - name: PolyMC Matrix Support Room
-    url: https://matrix.to/#/#support:polymc.org
-    about: Please ask for support here before opening an issue.
+# contact_links:
+#   - name: Prism Launcher Matrix Support Room
+#     url: about:blank # Add proper link in the future
+#     about: (⚠ Room not created yet ⚠) Please ask for support here before opening an issue.
```
Comment (RuiNtD) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
contact_links:
  - name: Prism Launcher Matrix Support Room
    url: https://matrix.to/#/#prism-support:matrix.org
    about: Please ask for support here before opening an issue.
```
Everything else looks good to me.

Comment (TheOPtimal) [reply .github/ISSUE_TEMPLATE/config.yml]: We could potentially also add the Discord server, but ideally we want them on Matrix

**Review by RuiNtD — APPROVED** [review]
Looks good to me. 👍🏻 

EDIT: As for adding a Discord link, that'd be better off on a separate PR anyways, since it's doesn't belong with rebranding.

**Comment by DioEgizio** [comment]
Please sign off your commits, otherwise lgtm

**Comment by MenacingPerson** [comment]
Here's the commands for a quick signoff

```sh
git rebase upstream # assumes you have remote upstream as this repo
git rebase -i HEAD~7 --signoff # squash everything
git push -f
```

**Comment by Scrumplex** [comment]
Maybe just squash all commits into one and sign that off

**Comment by TheOPtimal** [comment]
whoever merges this needs to squash commits, i forgot how to do it

**Review by Scrumplex — APPROVED** [review]

**Review by DioEgizio — APPROVED** [review]
looks good

**Pull request merged**